src/simply-rets-api-helper.php

```diff
@@ -1046,7 +1046,7 @@ public static function srResidentialDetailsGenerator( $listing ) {
         $next_openhouses = $upcoming_openhouses > 0 ? $openhouses : NULL;
 
         $next_openhouses_banner = "";
-        if ($next_openhouses) {
+        if ($has_openhouses && $next_openhouses) {
```

Comment:
This is the actual fix, somehow this check got dropped off in later commits from #137